test/unit-tests/openTab.test.js

```diff
@@ -26,6 +26,10 @@ describe('openTab', () => {
     taiko.__set__('_client', new EventEmitter());
   });
 
+  after(() => {
+    taiko = rewire('../../lib/taiko');
```

Comment:
this is done on the `before` - why do it in `after`?

Reply:
rewire does not have a reset option available... calling that again resets the updated values https://github.com/jhnns/rewire/issues/71